service/dap/server.go

```diff
@@ -1546,3 +1595,17 @@ func (s *Server) doCommand(command string) {
 			}})
 	}
 }
+
+func (s *Server) toLocalPath(path string) string {
```

Comment:
Seeing these in the code, I am more convinced now that `toClientPath` and `toServerPath` would be more clear.

Reply:
Will make the change.